Modules/getpath.c

```diff
@@ -775,6 +775,14 @@ library_to_dict(PyObject *dict, const char *key)
 }
 
 
+PyObject *
+_Py_GetGetPathCodeObject()
```

Comment:
Looks like a typo.

```suggestion
_Py_GetPathCodeObject()
```
The same for all calls of this function.